core/pom.xml

```diff
@@ -13,4 +13,13 @@
     <groupId>com.github.szalyi</groupId>
     <artifactId>flashcard-app-core</artifactId>
     <version>1.0-SNAPSHOT</version>
+
+
+    <dependencies>
+        <dependency>
+                <groupId>javax.persistence</groupId>
+                <artifactId>persistence-api</artifactId>
+                <version>${perersistence.api.version}</version>
```

Comment:
1. Typo in property name.
2. Use `<dependencyManagement>` to define dependencies in the parent module and use them in modules accordingly.